origin-growth/src/enums.js

```diff
@@ -0,0 +1,45 @@
+class Enum extends Array {
+  constructor(...args) {
+    super(...args)
+
+    for(const k of args) {
+      this[k] = k
+    }
+  }
+
+}
+
+const GrowthEventTypes = new Enum(
+  'ProfilePublished',
+  'EmailAttestation',
+  'FacebookAttestation',
+  'AirbnbAttestation',
+  'TwitterAttestation',
+  'PhoneAttestation',
+  'ListingCreated',
+  'ListingPurchased'
+)
+
+const GrowthRewardStatuses = new Enum(
+  'Pending',
+  'Fraud,',
+  'Blocked',
+  'Paid'
+)
+
+const GrowthInviteContactTypes = new Enum(
```

Comment:
I think this should be: 
```
new Enum(
    'Email',
    'Unknown'
)
```

Some thoughts about this here: https://docs.google.com/document/d/1QN9xf-4jIEdMyTyOZc2ocJHAjPCgSNwFGAVdL4ZfyZM/edit?ts=5c480f25#heading=h.1ewnimab3d0h

Reply:
I added SMS because that's something Matt ask me if we can support in the future.
Regarding Unknown, I added a comment in the design doc. I'm not sure yet we need to populate this table if the referee comes from an unknown source... Let's chat.

Reply:
👍 added as an agenda item in our [doc](https://docs.google.com/document/d/15nzudW56hQVIWHc7DRJI5XzxenM0-oc2p7z4vOqc8ms/edit?ts=5b7b483f#)